.circleci/config.yml

```diff
@@ -829,7 +829,7 @@ jobs:
   check-gradle-dependencies:
     executor:
       name: ubuntu
-      machine: small
+      machine: xlarge
```

Comment:
```suggestion
      machine: large
```
or even `medium`

Reply:
Ah, I didn't realize that we're using https://circleci.com/docs/configuration-reference/#linuxvm-execution-environment,  I thought we're using docker ones: https://circleci.com/docs/configuration-reference/#docker-execution-environment

Reply:
Ok, so we're actually using docker ones. So let's keep `xlarge` for now.